labelbox/schema/project.py

```diff
@@ -420,15 +433,33 @@ def export_v2(self,
             "label_details": False
         })
 
+        _filters = filters or ProjectExportFilters({
+            "last_activity_at": None,
+            "label_created_at": None
+        })
+
+        def _get_timezone() -> str:
+            timezone_query_str = """query CurrentUserPyApi { user { timezone } }"""
```

Comment:
again, this is prob more for my own education, but don't we have some sort of sdk-side authenticated user object that should already expose time sone?

Reply:
I've looked into the codebase myself and asked @kkim-labelbox about that - looks like we haven't had an use case to fetch timezone before.